change/@ni-nimble-angular-fae5ec22-a3d0-4c1f-924b-e36a0f9b9f27.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Switch from deprecated Sass @import to @use",
+  "packageName": "@ni/nimble-angular",
+  "email": "7282195+m-akinc@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
Looks like some references to `@import` in the SCSS context were missed in the Blazor README